paddle/fluid/operators/listen_and_serv_op.h

```diff
@@ -12,11 +12,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
+#ifndef PADDLE_FLUID_OPERATORS_LISTEN_AND_SERV_OP_H_
+#define PADDLE_FLUID_OPERATORS_LISTEN_AND_SERV_OP_H_
```

Comment:
We use `#pragma once` instead, that's the style which differs from google style.

Reply:
Thanks, I will fix this in next commit.